util/data_set_helpers.py

```diff
@@ -0,0 +1,61 @@
+
+import threading
+import numpy as np
+import tensorflow as tf
+
+from util.text import ctc_label_dense_to_sparse
+
+class DataSets(object):
+    def __init__(self, train, dev, test):
+        self._dev = dev
+        self._test = test
+        self._train = train
+
+    def start_queue_threads(self, session):
+        self._dev.start_queue_threads(session)
+        self._test.start_queue_threads(session)
+        self._train.start_queue_threads(session)
+
+    @property
+    def train(self):
+        return self._train
+
+    @property
+    def dev(self):
+        return self._dev
+
+    @property
+    def test(self):
+        return self._test
+
+class SwitchableDataSet(object):
+    def __init__(self, data_sets):
+        self._data_sets = data_sets
+        self._sets = [data_sets.train, data_sets.dev, data_sets.test]
+        self._queues = [s._example_queue for s in self._sets]
```

Comment:
We should expose the example queue in the DataSet instead of poking at the internals here.